lib/config/rules/babel.js

```diff
@@ -4,10 +4,10 @@
 module.exports = {
   // Ignores capitalized decorators (@Decorator)
   'babel/new-cap': ['error', {newIsCap: true, capIsNew: false}],
-  // guard against awaiting async functions inside of a loop
-  'babel/no-await-in-loop': 'off',
```

Comment:
Shouldn't we be adding the [equivalent eslint rule](http://eslint.org/docs/rules/no-await-in-loop)?